tests/features/search/querying.feature

```diff
@@ -40,7 +40,7 @@ Feature: Search querying
 
   Scenario: No auto suggestion
     Given I am on the `search page`
-    And I enter "World" in the `search box`
+    And I enter "Hundred" in the `search box`
```

Comment:
The point of this specific scenario was to ensure the autosuggest feature was behind a feature toggle that's off by default, so the thing being searched for should be something that would yield results.

Reply:
thanks for explaining, have reverted